test/standalone/Vegetation/plant_hydraulics_test.jl

```diff
@@ -307,7 +307,7 @@ for FT in (Float32, Float64)
                                     vcat(root_depths, [0.0])[2:end] -
                                     vcat(root_depths, [0.0])[1:(end - 1)]
                                 ),
-                            ) * AI[:root]
+                            ) * AI[:stem]
```

Comment:
why did this change from root to stem?

Reply:
In our src code, we use the above ground area index here, not the root area index. This was making sure the two align since this test is testing the src function.

Reply:
makes sense, thank you!